README.md

```diff
@@ -25,11 +25,12 @@ are best understood by tax attorneys, corporate lawyers, and other professionals
 Decisions related to [negotiating an offer](#offers-and-negotiations) and
 [exercising stock options](#stock-option-scenarios), in particular, can have **major financial
 consequences**. Because the value of employee equity is determined by the fate of the
-company, when a company does not succeed, an employee’s equity may be
-[worth nothing](#what-is-private-stock-worth), and the costs of exercising may not be
-recouped. Even when a company does succeed, an employee may suffer
-[catastrophic tax pitfalls](#tax-dangers) because they didn’t anticipate the tax consequences
-of their decisions.
+company, an employee’s equity may be [illiquid](#what-is-private-stock-worth) for a long time
```

Comment:
Probably worth defining "illiquid" in "Vesting and liquidity" or in "Can you sell private stock?" We use the term once (other than here!), in "Taxes on RSUs" and the meaning may not be clear to everyone. Or even here, "an employee's equity may be illiquid--that is, unable to be sold for cash--for a long time..."